src/components/Pages/HomeSections/videoSection/VideoSection.tsx

```diff
@@ -1,36 +1,165 @@
+                    >
+                        <iframe
+                            src={`https://www.youtube.com/embed/qLznfpgj1E4?wmode=transparent&autoplay=1`}
+                            title="YouTube video player"
+                            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                            loading="lazy"
+                            allowFullScreen
+                            className={
+                                isVideoPlaying
+                                    ? `${styles.videoPlayer} ${styles.videoImage}`
+                                    : styles.none
+                            }
+                        ></iframe>
+                        <Image
+                            alt="videoImage"
+                            src={videoSectionImage}
+                            quality={100}
+                            className={
+                                isVideoPlaying ? styles.none : styles.videoImage
+                            }
+                            onClick={() => handlePlayVideo()}
+                        />
+                        <div
+                            className={
+                                isVideoPlaying
+                                    ? styles.none
+                                    : styles.textWrapper
+                            }
+                        >
+                            <p className={styles.timerP}>
+                                {currentLang["explained in 90 second"]}
+                            </p>
+                            <h1 className={styles.header}>
+                            {currentLang["How does FreeDi work?"]}
+                            </h1>
+                            <p className={styles.underHeaderP}>
+                            {currentLang["Let us walk you through how our app works"]}
+                            </p>
+                        </div>
+                        <div
+                            className={
+                                isVideoPlaying
+                                    ? styles.none
+                                    : styles.logoWrapper
+                            }
+                        >
+                            <FreediLogoIcon size={"1.9rem"} color={"#fff"} />
+                            <FreediIcon
+                                height="1.9rem"
+                                width="4.1rem"
+                                freeColor="#FFE601"
+                                diColor="#fff"
+                            />
+                        </div>
+                        <Image
+                            alt="videoPeopleImage"
+                            src={videoSectionPeopleImage}
+                            quality={100}
+                            className={styles.videoPeopleImage}
+                        />
+                    </div>
+                </>
```

Comment:
A fragment with only one child is redundant.
Remove <></> tag.

Reply:
done